### ISSUE TITLE
without RT_Thread

### ISSUE THREAD

**Issue by AndreQingyuWu** [majianjia/nnom#171]
can nnom run correctly in MCU SDK without RT_Thread, for example running in nrf5-sdk

**Comment by majianjia** [comment]
Yes it can. It is an independent package that doesnt rely on any RTOS. 
You can check the [CI example for running on PC](https://github.com/majianjia/nnom/tree/master/examples/auto_test) or the recently [submitted contest that runs on STM32 HAL without RTOS using static memory allocation](https://github.com/majianjia/tinyml-contest)

**Comment by AndreQingyuWu** [comment]
Thank you very much. I will try to apply nnom to a low power IoT project based on nrf52833. Pure C is very valuable!

**Comment by jonnor** [comment]
This has been answered, and should be closed. CC @majianjia
